pkg/api/interceptor.go

```diff
@@ -62,14 +75,20 @@ func (wa *WalletAuthorizer) Stream() grpc.StreamServerInterceptor {
 		info *grpc.StreamServerInfo,
 		handler grpc.StreamHandler,
 	) error {
-		// TODO(mk): Add metrics
+		if wa.Ratelimits {
```

Comment:
It may make sense to have completely different limits for stream requests given that they are long lasting and tax the system differently.